src/main/java/mate/academy/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -43,6 +43,11 @@ public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundExcept
         return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(RegistrationException exception) {
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.CONFLICT);

```

Comment:
if these return plain text, we should change it to some json with timestamp, status phrase and message fields